react/DeviceDetector.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import useDevice, { DeviceInfo } from './useDevice'
+
+const DeviceDetector = ({
```

Comment:
nitpick: DeviceDetector still is a FunctionalComponent, I'd make it say it returns `: FC<Props> = `